src/scripts/main.js

```diff
@@ -1 +1,64 @@
 'use strict';
+
+const getElement = ([message, className]) => {
+  const element = document.createElement('div');
+
+  element.setAttribute('data-qa', 'notification');
+  element.innerText = message;
+  element.className = className;
+
+  return document.body.appendChild(element);
+};
+
+let leftClick = false;
+let rightClick = false;
+
+const promise1 = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    leftClick = true;
+
+    if (leftClick) {
+      resolve(['First promise was resolved', 'success']);
+    }
+  });
+
+  if (!leftClick) {
+    setTimeout(() => {
+      // eslint-disable-next-line prefer-promise-reject-errors
+      reject(['First promise was rejected', 'error']);
+    }, 3000);
+  }
```

Comment:
The timeout logic for rejecting `promise1` is still outside the event listener. This means that the rejection will occur after 3 seconds regardless of whether a click event has resolved the promise. Consider moving this logic inside the event listener or using a different approach to ensure the promise is only rejected if it hasn't been resolved within the timeout period.